config/config.go

```diff
@@ -41,10 +41,10 @@ func FindConfig(fs afero.Fs, configFile string) ([]byte, int, error) {
 	if err != nil {
 		return nil, 0, errs.WrapUser(err, "unable to open config file")
 	}
-	reader := io.ReadCloser(f)
-	defer reader.Close()
 
-	b, e := ioutil.ReadAll(reader)
+	b, e := ioutil.ReadAll(f)
+	defer f.Close()
```

Comment:
thanks! but maybe move it right after line 43 so it is next to the place where you create the file (and it is easier for readers to keep track of)

Reply:
ping